<launch>
    <node pkg="pladypos" name="pladypos_node" type="pladypos_node">
      <param name="PortName" value="/dev/ttyS1" />
      <param name="Baud" value="57600" />
      <param name="maxThrust" value="0.85"/>
      <param name="minThrust" value="-0.85"/>
      <remap from="tauIn" to="tauOut" />
      <!-- <remap from="tauAch" to="tauAchPl" /> -->
    </node>
</launch>
